netbsd/libs.md

```diff
@@ -0,0 +1,143 @@
+# Instructions for obtaining EFI related libraries on NetBSD
+
+Read [README.md](./README.md) first for general information.
+
+NetBSD already has support for EFI variables in the kernel, but this
+functionality is not yet available in releases, which is why we need to build
+the kernel ourselves.
+
+The plan is:
+ * install kernel with support for EFI variables
+ * prepare environment for building ports
+ * build and install `efivar` port
+ * build a test tool
+ * verify that it works
+
+## Install kernel with support for EFI variables
+
+You can either cross-compile and make an ISO as described in the README or run
+the commands below as `root` on an already running NetBSD:
+
+```
+# install git
+export PATH="/usr/pkg/sbin:/usr/pkg/bin:$PATH"
+export PKG_PATH="http://cdn.netbsd.org/pub/pkgsrc/packages/NetBSD/amd64/9.3/All/"
+pkg_add git
+
+# clone sources
+export GIT_SSL_NO_VERIFY=true
+git clone -b os-interface --depth 1 https://github.com/3mdeb/NetBSD-src.git /usr/src
+
+# build the kernel
+cd /usr/src/sys/arch/amd64/conf
+cp GENERIC MYKERNEL
+mkdir /usr/obj
+cd /usr/src
+./build.sh tools
+./build.sh -u -j$(sysctl -n hw.ncpu) kernel=MYKERNEL
+
+# install as a second kernel to be used when needed
+cp /usr/obj/sys/arch/amd64/compile/MYKERNEL/netbsd /netbsd.my
+
+# create /dev/efi device file
+make -C /usr/src/etc MAKEDEV TOOLDIR=/usr/obj/tooldir.NetBSD-9.3-amd64/
```

Comment:
I'm getting an error at this point:

```
# make -C /usr/src/etc MAKEDEV TOOLDIR=/usr/obj/tooldir.NetBSD-9.3-amd64/
make: "/usr/src/etc/Makefile" like 374: Malformed conditional (${MKDTB} != "no")
make: Fatal errors encountered -- cannot continue
make: stopped in /usr/src/etc
```

Reply:
I've edited `Makefile` and forgot about this, adding `MKDTB=no` to the command works too.

Reply:
Thanks